components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,7 +195,53 @@
        })
 
        return optionalsWithValue
-     }
+     },
+     getFormatDate: function (date,precision){
+
+      let [day, month, year] = date.split('/');
+      day = Number(day);
+      month = Number(month)
+      year = Number(year);
+      let currentDate = new Date(year, month - 1, day);
+      
+      
+      //the precision of the dates can either be or 9 11(full day/month/year),10(month/year),9(year)
+      
+      // If we don't know the day, month, and year, then wikidata precision will be less than 11 and the dates don't need Julian conversion
+      if (precision === '10') {
+          // Then return the date with month and year only
+          let dateString = `${(currentDate.getMonth() + 1).toString().padStart(2, '0')}/` + `${currentDate.getFullYear()}`;
+          return dateString;
+      }  
+          // If only year is known
+      if (precision === '9')  {
+          // Then return the date with year only
+          let dateString = `${currentDate.getFullYear()}`;
+          return dateString;
+      }
+        
+      
+
+      //if we have the full date (day,month,year) them the date needs to be converted into Julian
+      const cutoffDate = new Date(1700,3,1) // the date where the difference between julian and gregorian cjanges from 10 to 11
```

Comment:
It might be worth highlighting this is the gregorian date. I got confused for a wee bit thinking it was the julian date. My bad as it is fairly obvious in hindsight but might save other folks time. Thank you

Reply:
Changes made. Thanks Ruby